src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write code here
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+
+spider.style.left = wall.clientWidth / 2 - spider.clientWidth / 2 + 'px';
+spider.style.top = wall.clientHeight / 2 - spider.clientHeight / 2 + 'px';
```

Comment:
Would be better to move these values to variables and then add `px` to improve the code readability 
```suggestion
spider.style.left = `${x}px`;
spider.style.top = `${y}px`;
```